packages/core-data/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+	"rootDir": "src",
+	"declarationDir": "build-types",
+	"noUnusedParameters": false
+  },
+  "references": [
+  ],
+  "include": [
+	"src/types/**/*",
+	"src/*.ts"
```

Comment:
```suggestion
	"src/**/*.ts"
```

Reply:
Thanks! I removed the tsconfig.json for now, but this will come handy later on